tests/test_tree.py

```diff
@@ -10,6 +11,8 @@
 
 @pytest.mark.parametrize("root_name", [None, "root"])
 def test_tree(root_name: Any) -> None:
+    os.environ["OVERRIDE_COLOR_SYSTEM"] = "truecolor"
```

Comment:
Just out of interest, why's this needed?

Reply:
`test_tree.py` expects the tree to be rendered with the control sequences for text formatting (e.g. bold font). However, by default `rich` doesn't include that when running within pytest, because it detects that it is not running in a real terminal as the output is captured by pytest. This overrides the behavior in `rich`. This issue only surfaced now, because I added `rich` to the test dependencies and therefore `test_tree.py` wasn't skipped anymore.
https://github.com/zarr-developers/zarr-python/blob/main/tests/test_tree.py#L31-L54